pallene/coder.lua

```diff
@@ -967,8 +969,15 @@ gen_cmd["Binop"] = function(self, cmd, _func)
     -- For integer division and modulus:
     local function int_division(fname)
         local line = cmd.loc.line
-        return (util.render([[ $dst = $fname(L, $x, $y, $line); ]], {
-            fname = fname, dst = dst, x = x, y = y, line = C.integer(line) }))
+        print(require("inspect")(cmd))
```

Comment:
Please review the diff when you create the PR to avoid leaving behind these print statements.

Reply:
I remove the print statements before we merge, I find them useful. :sweat_smile: